.github/workflows/main.yml

```
@@ -23,8 +23,8 @@ jobs:
         pip install -U -r requirements-dev.txt sphinx-intl qiskit
     - name: Build and publish
       env:
-        encrypted_rclone_key: {{ secrets.encrypted_rclone_key }}
-        encrypted_rclone_iv: {{ secrets.encrypted_rclone_iv }}
-        TRANSLATION_LANG: {{ matrix.language }}
+        ENCRYPTED_RCLONE_KEY: ${{ secrets.encrypted_rclone_key }}
```

this needs to be lower case because the deploy script is using $encrypted_rclone_key
```suggestion
        encrypted_rclone_key: ${{ secrets.encrypted_rclone_key }}
```